### ISSUE TITLE
Accessibility: Add proper access-keys to controls

### ISSUE THREAD

**Issue by jonthysell** [jonthysell/Chordious#68]
Keyboard users should be able to easily hotkey around the UI without having to tab through everything manually

**Comment by jonthysell** [comment]
All that remains are the windows that modify DiagramStyle since there are so many elements and navigating by keyboard to each one may not be feasible.
